examples/index.ts

```diff
@@ -2,6 +2,7 @@ import Phoenix from "../src/index";
 
 const start = async () => {
   const phoenix = new Phoenix({
+    username: "phoenix",
     token: "your_token_goes_here",
     host: "http://127.0.0.1:9740",
   });
```

Comment:
Add optional comment:

```suggestion
  const phoenix = new Phoenix({
    username: "phoenix", // optional
    token: "your_token_goes_here",
    host: "http://127.0.0.1:9740", // optional
  });
```